### PR TITLE
fixes various console errors

### PR DESCRIPTION
**Pull request by jack-carmichael**
These are errors shown in the dev environment that should be fixed from this PR.

Screenshot 2023-07-25 at 18 34 49
Screenshot 2023-07-25 at 18 34 36
Screenshot 2023-07-25 at 18 34 31
Screenshot 2023-07-25 at 18 34 17

The invalid nesting throws an error on prod.

Screenshot 2023-07-25 at 18 32 13

### CODE REVIEW

**Comment by FB-Reda** [comment]
king

File: components/headerNav/index.tsx
Comment (FB-Reda) [inline components/headerNav/index.tsx:1]: This change gets rid of pointer behavior, would it make sense to keep the Logo styled-component and instead have that wrap the Next/Image Component?

File: components/footer/index.tsx
Comment (FB-Reda) [inline components/footer/index.tsx:1]: Powered by vercel lost its centering

File: app/components/staff.app.tsx
Comment (FB-Reda) [inline app/components/staff.app.tsx:1]: altered "letter component" spacing looks suboptimal on mobile widths

**Review by FB-Reda — CHANGES_REQUESTED** [review]

File: components/headerNav/index.tsx
Comment (jack-carmichael) [reply components/headerNav/index.tsx:1]: removed legacyBehavior which fixed the pointer behavior

File: components/footer/index.tsx
Comment (jack-carmichael) [reply components/footer/index.tsx:1]: thought I was removing rogue divs, turns out they were needed ¯\_(ツ)_/¯

File: app/components/staff.app.tsx
Comment (jack-carmichael) [reply app/components/staff.app.tsx:1]: I think I fixed this? it looks closer now to the live site

**Review by FB-Reda — CHANGES_REQUESTED** [review]
Still looking for tweaks on the letter component

- Would prefer the margins match the other pages.  I think this is because we're removing the wrapping div so there's 16px on each side unaccounted for.  Maybe this should be simplified as larger padding on the Card idk
- The paragraph text was previously center justified whereas now it's left aligned.

**Review by FB-Reda — APPROVED** [review]
Good work, thanks!

**Pull request merged**